R/implicit_integer_linter.R

```diff
@@ -30,18 +37,28 @@
 #'   linters = implicit_integer_linter()
 #' )
 #'
+#' lint(
+#'   text = "1:10",
+#'   linters = implicit_integer_linter(allow_colon = TRUE)
+#' )
+#'
 #' @evalRd rd_tags("implicit_integer_linter")
 #' @seealso [linters] for a complete list of linters available in lintr.
 #' @export
-implicit_integer_linter <- function() {
+implicit_integer_linter <- function(allow_colon = FALSE) {
```

Comment:
Shouldn't the default argument be `TRUE` for this parameter? 
Which cases that should produce a lint will be missed out on, if we default to `TRUE`?

Reply:
Done for back-compatiblity... current HEAD implicitly uses `FALSE`.

I'd be open to changing it later, but at least for this release we should use this default.

Reply:
SGTM.

Can you please file a new issue to remember to do this? And maybe also choose the appropriate milestone for the issue?

Reply:
#1694. Let's hold off on choosing a milestone, but maybe 3.1.0? Let's see if we get any feedback after 3.0.2 (I guess it's really 3.0.3 now) is released with this PR.